Elizabeth_Theme/src/css/_custom.scss

```diff
@@ -0,0 +1,79 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+@import 'colors';
+@import 'header/navigation.scss';
+@import 'footer/footer.scss';
+@import 'welcome_page/welcome_page.scss';
+
+body {
+  //BUTTONS
+  a.btn-orange {
```

Comment:
CSS class names should be generic enough to where they can be reused, but not tied to a color or a specific use case.